### ISSUE TITLE
Small town in Ireland has the shape of a perfect ellipse.

### ISSUE THREAD

**Issue by dianashk** [whosonfirst/whosonfirst-data#234]
This seems to be incorrect. Not sure how to get the right geometry for this, but at the very least we should make it a point so as not to provide bad admin-lookup information.

https://whosonfirst.mapzen.com/spelunker/id/101873779/

![whosonfirst-spelunker-1459200999291](https://cloud.githubusercontent.com/assets/5446511/14091057/e792b92a-f50b-11e5-87d3-0e63c4b6ccec.png)

Connected to pelias/pelias#292

**Comment by bcamper** [comment]
:o:

**Comment by thisisaaronland** [comment]
This is not really a bug, per se. It's a property of the QS data that has always been present. For places without a known and confirmed polygon an ellipse was calculated from a centroid for that location.

It is essentially the same a bounding box for a place but with all those annoying false-positive corners shaved off.

**Comment by thisisaaronland** [comment]
Also, screenshots with geometries!

**Comment by nvkelso** [comment]
Funny enough there are some legit towns (some in the US state of Georgia come to mind) where they are a perfect eclipse. 

@dianashk would you want to make it a point, though? An alternative is for us to mark that these are buffered point polygons (that's buried in the QS metadata properties now). Then during reverse geo lookup you could sort these behind the other localities / similar features of same place type so it would be the last feature queried. If there's better data then that's returned, else the coarse geom here is returned?

**Comment by dianashk** [comment]
The screenshot feature is very convenient! Thanks @thisisaaronland!

I would think we never want to use an approximated geometry in admin lookup. We can still import the place as its own record and just leave it a point. We do that with all of geonames.

At the very least, we'd need a flag, as @nvkelso said, to let us know the geometry is approximated, so we can *do the right thing* about it.

I don't see any value in the ellipse. Just out of curiousity, are there other use-cases where it's better to have the ellipse than a point?

**Comment by thisisaaronland** [comment]
A flag would be useful. At some point after the remainder of WOE/GeoPlanet is imported we will probably assign a derived geometry which will be less-correct than accurate but also less-incorrect than the entire planet.

`mz:is_derived=1` ?

**Comment by nvkelso** [comment]
Conveniently QS stores this explicitly, and WOF preserves it as:

`qs:type = buffered point`

@dianashk Can you explain more why you think this is a regression related to WOF adoption? This feature hasn't changed.

**Comment by dianashk** [comment]
You're right, it's not a regression caused by the WOF data. It was exposed during the integration because we changed how we generate labels and because our previous setup didn't have `macroregions`. Now we use `macroregions` in our labels and still aren't able to lookup the right `locality`, which is what should really be part of the label. 

With that said, it's not something that would hold up the release, but it's something we should think about fixing, especially since the next step after integration is to start returning geometries via the `/place` endpoint.

**Comment by dianashk** [comment]
If the flag from QS is already in the WOF data, we can just use that for now, so there might not be any action needed on the data side.

**Comment by nvkelso** [comment]
WOF should provide you an attribute for this, you shouldn't dip into a QS property for it.

**Comment by nvkelso** [comment]
Similar to recent work that unset the junk polygon for neighbourhoods in the USA, I'm fine with unsetting this to the point instead and moving the old geom to alt file.
